content/en/tracing/trace_collection/trace_context_propagation/_index.md

```diff
@@ -29,6 +29,7 @@ By default, the Datadog SDK extracts and injects distributed tracing headers usi
 
 - [Datadog][1] (takes higher precedence when extracting headers)
 - [W3C Trace Context][2]
+- [Baggage][7]
```

Comment:
This link isn't working because the reference for `[7]` is within the `Java` tab. It will need to be moved to the list of reference links at the very bottom, outside of any tabs, on line `651`.